src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.NetStandard.cs

```diff
@@ -160,17 +164,16 @@ void PositionImage(Image imageElement)
 		{
 			try
 			{
-				var manifestFilename = "appmanifest.js";
 				string? manifestString = default;
 				try
 				{
-					var storageFile = await StorageFile.GetFileFromApplicationUriAsync(new Uri($"ms-appx:///{manifestFilename}"));
+					var storageFile = await StorageFile.GetFileFromApplicationUriAsync(new Uri($"ms-appx:///{WasmAppManifestFilename}"));
 					manifestString = await FileIO.ReadTextAsync(storageFile);
```

Comment:
instead of try and fail, we can maybe check StorageFile.IsAvailable?

Reply:
That property is an illusion ;-) IsAvailable is an instance property which means you need a StorageFile. The only way to get a StorageFile is using code similar to what I have here, which throws an exception if the file doesn't exist..... awesome, just awesome Microsoft!